### ISSUE TITLE
Help Icon content in the modal dialog versions sometimes inherits unwanted CSS layout (example: Help Icons in tables)

### ISSUE THREAD

**Issue by danowar2k** [moodleuulm/moodle-theme_boost_campus#106]
We detected something minor with modal help icon dialogs. Those dialogs are placed inside elements near the help icons.

When the help icon is inside a table header, the help icon dialog inherits text layout (e.g. text alignment, font weight, etc.) which can lead to unwanted side effects on the text inside the dialog.

An example for this can be seen at mod/forum/index.php.

**Comment by danowar2k** [comment]
![image](https://user-images.githubusercontent.com/10991509/112617716-9afae480-8e25-11eb-9341-991c195bebad.png)

**Comment by danowar2k** [comment]
Locally we fixed this with a CSS rule for this specific help dialog (we don't know yet where else this could be affecting the layout).

```
/**
The help icon is inside a table where text is centered and bolder
This fixes both on mod/forum/index.php
 */
#mod_forum-emaildigesttype-modal-dialogue .modal-body {
  text-align: left;
  font-weight: initial;
}
```

If we knew all the places where this happens we would probably be more unspecific with the rule...

```
/**
Alternative way for all modal help icon dialog in table headers
 */
th .modal-dialog .modal-body {
  text-align: left;
  font-weight: initial;
}
```

But maybe the help icon modal dialogs should receive their own Boost Campus CSS class so that rules could be directed at those specific dialogs?

**Comment by danowar2k** [comment]
PR: https://github.com/moodleuulm/moodle-theme_boost_campus/pull/107

**Comment by abias** [comment]
This has been solved by #107
